### PR TITLE
update download and index pages to make them reflect current state

### PR DESCRIPTION
**Pull request by OsirisTerje**

### CODE REVIEW

**Review by Arhell — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -64,9 +64,10 @@ <h2><i class="fas fa-bullhorn fa-sm"></i> NUnit is a part of the .NET Foundation
                 <p>
                   The NUnit Project is a member of the <a href="https://www.dotnetfoundation.org">.NET Foundation</a>.
                   NUnit is run by the core team, <a href="https://www.alteridem.net">Rob Prouse</a>,
-                  <a href="https://www.charliepoole.org/">Charlie Poole</a>, <a href="https://Hermit.no">Terje Sandstrom</a>,
+                  <a href="https://Hermit.no">Terje Sandstrom (Lead)</a>,
                   <a href="https://github.com/jnm2">Joseph Musser</a>
-                  and <a href="https://github.com/mikkelbu">Mikkel Nylander Bundgaard</a>.
+                  <a href="https://github.com/mikkelbu">Mikkel Nylander Bundgaard</a>,
+                  and <a href="https://github.com/stevenaw">Steven Weerdenburg</a>
```
Comment (SeanKilleen) [inline index.html:70]:
```suggestion
                  <a href="https://github.com/stevenaw">Steven Weerdenburg</a>,
                  and <a href="https://github.com/SeanKilleen">SeanKilleen</a>
```

File: index.html [hunk 2]
```diff
@@ -64,9 +64,11 @@ <h2><i class="fas fa-bullhorn fa-sm"></i> NUnit is a part of the .NET Foundation
                 <p>
                   The NUnit Project is a member of the <a href="https://www.dotnetfoundation.org">.NET Foundation</a>.
                   NUnit is run by the core team, <a href="https://www.alteridem.net">Rob Prouse</a>,
-                  <a href="https://www.charliepoole.org/">Charlie Poole</a>, <a href="https://Hermit.no">Terje Sandstrom</a>,
+                  <a href="https://Hermit.no">Terje Sandstrom (Lead)</a>,
                   <a href="https://github.com/jnm2">Joseph Musser</a>
-                  and <a href="https://github.com/mikkelbu">Mikkel Nylander Bundgaard</a>.
+                  <a href="https://github.com/mikkelbu">Mikkel Nylander Bundgaard</a>,
+                  <a href="https://github.com/stevenaw">Steven Weerdenburg</a>,
+                  and <a href="https://github.com/SeanKilleen">SeanKilleen</a>
```
Comment (SeanKilleen) [inline index.html:71]:
```suggestion
                  and <a href="https://SeanKilleen.com">SeanKilleen</a>
```

Actually, I'll use my website.

**Review by SeanKilleen — APPROVED** [review]
LGTM 👍 I'll merge once I check the preview site properly reflects the small edits I made.

File: index.html [hunk 3]
```diff
@@ -64,9 +64,11 @@ <h2><i class="fas fa-bullhorn fa-sm"></i> NUnit is a part of the .NET Foundation
                 <p>
                   The NUnit Project is a member of the <a href="https://www.dotnetfoundation.org">.NET Foundation</a>.
                   NUnit is run by the core team, <a href="https://www.alteridem.net">Rob Prouse</a>,
-                  <a href="https://www.charliepoole.org/">Charlie Poole</a>, <a href="https://Hermit.no">Terje Sandstrom</a>,
+                  <a href="https://Hermit.no">Terje Sandstrom (Lead)</a>,
                   <a href="https://github.com/jnm2">Joseph Musser</a>
-                  and <a href="https://github.com/mikkelbu">Mikkel Nylander Bundgaard</a>.
+                  <a href="https://github.com/mikkelbu">Mikkel Nylander Bundgaard</a>,
+                  <a href="https://github.com/stevenaw">Steven Weerdenburg</a>,
+                  and <a href="https://SeanKilleen.com">Sean Killeen</a>
```
Comment (SeanKilleen) [inline index.html:71]:
```suggestion
                  and <a href="https://SeanKilleen.com">Sean Killeen</a>.
```

Add period back. I need coffee this morning.

**Pull request merged**